analysis_template/sandboxes/ml_tf.txt

```diff
@@ -0,0 +1,5 @@
+# version 1
```

Comment:
these sandboxes are outdated and can be removed

Reply:
Done.